pyravendb/data/indexes.py

```diff
@@ -98,7 +98,10 @@ def __init__(self, name, maps, configuration=None, **kwargs):
 
         # fields is a  key value dict. the key is the name of the field and the value is IndexFieldOptions
         self.fields = kwargs.get("fields", {})
-
+        
+        # set "OutputReduceToCollection"
+        self.outputreducetocollection = kwargs.get("outputreducetocollection", None)
```

Comment:
```suggestion
        self.output_reduce_to_collection = kwargs.get("output_reduce_to_collection", None)
```

I think this should match the naming convention of the other args